docs/tables/aws_securityhub_insight.md

```diff
@@ -0,0 +1,57 @@
+# Table: aws_securityhub_insight
+
+An AWS Security Hub insight is a collection of related findings. It identifies a security area that requires attention and intervention. For example, an insight might point out EC2 instances that are the subject of findings that detect poor security practices. An insight brings together findings from across finding providers.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  insight_arn,
+  group_by_attribute,
+  region
+from
+  aws_securityhub_insight;
+```
+
+### List insights by a particular attribute
+
+```sql
+select
+  name,
+  insight_arn,
+  group_by_attribute,
+  region
+from
+  aws_securityhub_insight
+where
+  group_by_attribute = 'ResourceId';
+```
+
+### Get workflow status details for a particular insight
+
+```sql
+select
+  name,
+  insight_arn,
+  filters ->> 'WorkflowStatus' as workflow_status
```

Comment:
Are there any other commonly used or interesting properties we can extract out of `filters`? Looking at https://docs.aws.amazon.com/securityhub/1.0/APIReference/API_GetInsights.html, it seems like there are a lot of potential properties in `filters`, so are we able to expand this example (and possibly the one below)?